worlds/raft/__init__.py

```diff
@@ -37,16 +37,17 @@ class RaftWorld(World):
     lastItemId = max(filter(lambda val: val is not None, item_name_to_id.values()))
 
     location_name_to_id = locations_lookup_name_to_id
-    option_definitions = raft_options
+    options_dataclass = RaftOptions
+    options: RaftOptions
 
     required_client_version = (0, 3, 4)
 
     def create_items(self):
-        minRPSpecified = self.multiworld.minimum_resource_pack_amount[self.player].value
-        maxRPSpecified = self.multiworld.maximum_resource_pack_amount[self.player].value
+        minRPSpecified = self.options.minimum_resource_pack_amount.value
+        maxRPSpecified = self.options.maximum_resource_pack_amount.value
         minimumResourcePackAmount = min(minRPSpecified, maxRPSpecified)
         maximumResourcePackAmount = max(minRPSpecified, maxRPSpecified)
-        isFillingFrequencies = self.multiworld.island_frequency_locations[self.player].value <= 3
+        isFillingFrequencies = self.options.island_frequency_locations.value <= 3
```

Comment:
```suggestion
        isFillingFrequencies = self.options.island_frequency_locations.value <= IslandFrequencyLocations.option_random_on_island_random_order
```
Would recommend importing option classes from your options file, and then comparing to the option choices rather than a magic number. Not 100% required, just recommended. This also goes with the other instances of this happening in this file.

Reply:
Another approach to consider is to put a method in the option class.
```python
        isFillingFrequencies = self.options.island_frequency_locations.is_filling_frequencies()
```

Reply:
I like both of these. I referenced the option values from `self.options.name.option_name`, if this is unsafe and I should use `ClassName.option_name` lmk